Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+pipeline {
```

Comment:
I think when this replaces the legacy jobs we'll probably want more than 5 but for now that should be fine 😄